.circleci/config.yml

```diff
@@ -88,7 +88,7 @@ jobs:
           name: Terraform fmt
           command: terraform fmt -check
       - install_ansible
-      - run: 
+      - run:
           name: Check Ansible Syntax
           command: ansible-playbook -i '127.0.0.1,' ansible/prod.yml --syntax-check
   e2e_centos:
```

Comment:
It would be nice to create a RHEL test but it doesn't look like it's a supported blueprint for AWS LightSail (why do we use light sail anyways???)

```aws lightsail get-blueprints --region us-west-2```

Reply:
Yeah agreed. @sonnysideup set it up I think primarily because it was a straightforward / cost effective way of integration testing. Would love to know some alternatives (machine executor + vagrant?). RHEL might be difficult though in that it generally requires a subscription.